### PR TITLE
[XP-3079] Include .js and .ts file snowtype configuration options

### PR DESCRIPTION
**Pull request by DianaIgnatescu**

### CODE REVIEW

File: docs/collecting-data/code-generation/commands/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This page only summarizes the CLI commands and the options for each command. For
 
 ### `snowtype init`
 
-Initialize the setup of Snowtype code generation in a project. Creates the `snowtype.config.json` file.
```
Comment (igneel64) [inline docs/collecting-data/code-generation/commands/index.md:18]: Maybe we can change the `init` and `generate` statements to just `configuration file` versus adding all the options .

E.g:
`Initialize the setup of Snowtype code generation in a project. Creates the Snowtype configuration file.`

`Generates tracking code based on the Snowtype configuration file. Can generate/modify the `.snowtype-lock.json` file.`

WDYT ?

File: docs/collecting-data/code-generation/commands/index.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Checks for latest version updates in Data Structures and Event Specifications.
 
 ### `snowtype patch`
 
-Adds new Data Structures and Event Specifications in the `snowtype.config.json` file without needing to modify the file by hand.
+Adds new Data Structures and Event Specifications in the `snowtype.config.json`, `snowtype.config.js`, or `snowtype.config.ts` file without needing to modify the file by hand.
```
Comment (igneel64) [inline docs/collecting-data/code-generation/commands/index.md:43]: `patch` only works on JSON right ?

File: docs/collecting-data/code-generation/using-the-cli/index.md [hunk 3]
```diff
@@ -294,23 +294,23 @@ To add an Event Specification to the code generation, either manually or through
 
 ![event specification id](images/ts-id.png)
 
-Then you should add this ID to your `snowtype.config.json` `eventSpecificationIds` array.
+Then you should add this ID to your `snowtype.config.json`, `snowtype.config.js`, or `snowtype.config.ts` `eventSpecificationIds` array.
```
Comment (igneel64) [inline docs/collecting-data/code-generation/using-the-cli/index.md:297]: ^^ Same with the above, replacing all the possible file extensions with just `configuration file`.

File: docs/collecting-data/code-generation/commands/index.md (hunk 2 above)
Comment (DianaIgnatescu) [reply docs/collecting-data/code-generation/commands/index.md:43]: You're absolutely right, that should stay only for JSON. I'll sort it, thanks!

**Review by igneel64 — APPROVED** [review]

**Pull request merged**